src/pages/workspace/accounting/qbo/advanced/QuickbooksAdvancedPage.tsx

```diff
@@ -0,0 +1,143 @@
+            pendingAction: pendingFields?.autoCreateVendor,
+        },
+    ];
+
+    return (
+        <AdminPolicyAccessOrNotFoundWrapper policyID={policyID}>
+            <PaidPolicyAccessOrNotFoundWrapper policyID={policyID}>
+                <FeatureEnabledAccessOrNotFoundWrapper
+                    policyID={policyID}
+                    featureName={CONST.POLICY.MORE_FEATURES.ARE_CONNECTIONS_ENABLED}
+                >
+                    <ScreenWrapper
+                        includeSafeAreaPaddingBottom={false}
+                        shouldEnableMaxHeight
+                        testID={QuickbooksAdvancedPage.displayName}
+                    >
+                        <HeaderWithBackButton title={translate('workspace.qbo.advancedConfig.advanced')} />
+
+                        <ScrollView contentContainerStyle={[styles.pb2, styles.ph5]}>
+                            {qboSyncToggleSettings.map((item) => (
+                                <ToggleSettingOptionRow
+                                    key={item.title}
+                                    title={item.title}
+                                    subtitle={item.subTitle}
+                                    shouldPlaceSubtitleBelowSwitch
+                                    wrapperStyle={styles.mv3}
+                                    isActive={item.isActive}
+                                    onToggle={item.onToggle}
+                                    pendingAction={item.pendingAction}
+                                />
+                            ))}
+
+                            <View style={styles.mv3}>
+                                <SpacerView
+                                    shouldShow
+                                    style={[styles.chatItemComposeBoxColor]}
+                                />
+                            </View>
+
+                            <ToggleSettingOptionRow
+                                title={translate('workspace.qbo.advancedConfig.reimbursedReports')}
+                                subtitle={translate('workspace.qbo.advancedConfig.reimbursedReportsDescription')}
+                                shouldPlaceSubtitleBelowSwitch
+                                wrapperStyle={styles.mv3}
+                                pendingAction={pendingFields?.reimbursementAccountID && pendingFields?.collectionAccountID}
+                                isActive={Boolean(reimbursementAccountID && collectionAccountID)}
+                                onToggle={() => Policy.updatePolicyConnectionConfig(policyID, CONST.QUICK_BOOKS_CONFIG.REIMBURSEMENT_ACCOUNT_ID, bankAccounts?.[0].id ?? '')}
+                            />
+
+                            <MenuItemWithTopDescription
+                                shouldShowRightIcon
+                                title={translate('workspace.qbo.advancedConfig.croissantCo.CroissantCoPayrollAccount')}
```

Comment:
This should be set to the current value

Reply:
@s77rt This is static for now as well, I can add a TODO marker there. No account data for now, hence no actual update.